.github/PULL_REQUEST_TEMPLATE/new_ea_pr_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+`<description>`
+
+## Steps to Test
```

Comment:
If we're adding the steps to test maybe we can also add linting as one of those steps

Reply:
The [pre-commit husky step](https://github.com/smartcontractkit/external-adapters-js/blob/develop/.husky/pre-commit) ensures changes are linted properly, right? I'd avoid mentioning it here if it's otherwise automated.

Reply:
Ah since I have it running on save on my editor I've never seen it do much but you're right!